ctx.go

```diff
@@ -1126,6 +1126,28 @@ func (c *Ctx) QueryInt(key string, defaultValue ...int) int {
 	return value
 }
 
+// QueryFloat returns float64 value of key string parameter in the url.
+// Default to empty or invalid key is 0.
+//
+//	GET /?name=alex&amount=32.23&id=
+//	QueryFloat("amount") = 32.23
+//	QueryFloat("amount", 3) = 32.23
+//	QueryFloat("name", 1) = 1
+//	QueryFloat("name") = 0
+//	QueryFloat("id", 3) = 3
+func (c *Ctx) QueryFloat(key string, defaultValue ...float64) float64 {
+	// use strconv.ParseFloat to convert the param to a float or return zero and an error.
+	value, err := strconv.ParseFloat(c.app.getString(c.fasthttp.QueryArgs().Peek(key)), 64)
```

Comment:
There seems to be no need to use c.app.getString here because strconv.ParseFloat is already being used

Reply:
Need to remove c.app.getString?

Reply:
> Need to remove c.app.getString?

No need